integration/integrate-using-generic-webhooks.adoc

```diff
@@ -0,0 +1,42 @@
+[id="integrate-using-generic-webhooks"]
+= Integrating by using generic webhooks
+include::modules/common-attributes.adoc[]
+:context: integrate-using-generic-webhooks
+
+toc::[]
+
+[role="_abstract"]
+With {product-title}, you can send alert notifications as JSON messages to any webhook receiver.
+When a violation occurs, {product-title} makes an HTTP POST request on the configured URL.
+The POST request body includes JSON-formatted information about the alert.
+
+The webhook POST request's JSON data includes a `v1.Alert` object and any custom fields that you configure.
```

Comment:
It would be helpful to introduce the code example, either by adding to the intro sentence with something like, "...that you configure, as shown in the following example:" or else adding a code block title (`.Example webhook POST request`).